pkg/kubelet/util/pluginwatcher/plugin_watcher.go

```diff
@@ -161,14 +161,17 @@ func (w *Watcher) Stop() error {
 	close(w.stopCh)
 
 	c := make(chan struct{})
+	var once sync.Once
+	closeFunc := func() { close(c) }
```

Comment:
@mattjmcnaughton Add sync.Once to keep close() would be called only once

Reply:
Thanks :)

Reply:
close can be called on a chan as many times as you want. The once is not necessary.